capstone-app/src/app/modules/onboarding/onboarding.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatSelectModule } from '@angular/material/select';
+import { OnboardingComponent } from './onboard/onboarding.component';
+import { ContentComponent } from './content/content.component';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { MatDialogModule } from '@angular/material/dialog';
+
+@NgModule({
+  declarations: [OnboardingComponent, ContentComponent],
+  imports: [
+    CommonModule,
+    MatButtonModule,
+    FormsModule,
+    ReactiveFormsModule,
+    MatSelectModule,
+    NgbModule,
+    MatDialogModule,
+  ],
+  exports: [OnboardingComponent, ContentComponent]
+})
+export class OnboardingModule { }
```

Comment:
Did Onboarding require its own module? Why can't it just be a single component off of app.component.ts?

Reply:
there are specific imports that are only need for the onboarding or content components. So i think it would just be easier for it to have a module

Reply:
I prefer the one module per component style anyway.